services/blockchain-indexer/shared/indexer/tokenIndex.js

```diff
@@ -20,20 +20,33 @@ const {
 
 const { requestConnector } = require('../utils/request');
 
+const { isMainchain } = require('../chain');
+
 const config = require('../../config');
 
 const topLSKAddressesIndexSchema = require('../database/schema/topLSKAddresses');
 
 const MYSQL_ENDPOINT = config.endpoints.mysql;
 
+let CHAIN_ID;
+let LISK_TOKEN_ID;
+const LOCAL_ID_LSK = '00000000';
+
 const getTopLSKAddressesIndex = () => getTableInstance(
 	topLSKAddressesIndexSchema.tableName,
 	topLSKAddressesIndexSchema,
 	MYSQL_ENDPOINT,
 );
 
 const getLiskBalanceByAddress = async (address) => {
-	const LISK_TOKEN_ID = config.tokens.lisk.id;
+	if (!LISK_TOKEN_ID) {
+		if (!CHAIN_ID) {
+			const status = await requestConnector('getNetworkStatus');
+			CHAIN_ID = status.chainID;
```

Comment:
Could we destructure like below?
```suggestion
			{ chainID } = await requestConnector('getNetworkStatus');
```